### PR TITLE
Fix Test in Override Styles Using Important

### LINKED ISSUE #1337: Clickable "Next article" button in field guide.
Sometimes you want to leave an article to come back to and want to open the next article in a new tab, but the next button doesn't work that way, at least for now.

### LINKED ISSUE #8511: Fix Test in Override Styles Using Important
Challenge [Override All Other Styles by using Important](https://www.freecodecamp.com/challenges/override-all-other-styles-by-using-important#?solution=%0A%3Cstyle%3E%0A%20%20body%20%7B%0A%20%20%20%20background-color%3A%20black%3B%0A%20%20%20%20font-family%3A%20Monospace%3B%0A%20%20%20%20color%3A%20green%3B%0A%20%20%7D%0A%20%20%23orange-text%20%7B%0A%20%20%20%20color%3A%20orange%3B%0A%20%20%7D%0A%20%20.pink-text%7B%0A%20%20%20%20color%3A%20pink%20!important%3B%0A%20%20%7D%0A%20%20.blue-text%20%7B%0A%20%20%20%20color%3A%20blue%3B%0A%20%20%7D%0A%3C%2Fstyle%3E%0A%3Ch1%20id%3D%22orange-text%22%20class%3D%22pink-text%20blue-text%22%20style%3D%22color%3A%20white%22%3EHello%20World!%3C%2Fh1%3E%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.94 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```html

<style>
  body {
    background-color: black;
    font-family: Monospace;
    color: green;
  }
  #orange-text {
    color: orange;
  }
  .pink-text{
    color: pink !important;
  }
  .blue-text {
    color: blue;
  }
</style>
<h1 id="orange-text" class="pink-text blue-text" style="color: white">Hello World!</h1>

I can't go to the next challenge, because I haven't completed this point:

> Your pink-text class declaration should have the !important keyword to override all other declarations.

### PR DESCRIPTION
**Pull request by hangaebal**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (e.g. `Closes #XXXX`): Closes 
#### Description

closes #8511

### CODE REVIEW

**Comment by raisedadead** [comment]
@hangaebal Thanks for the PR.

You have not referenced the correct issue no :

image

Edit the PR description to reflect the correct Issue no.

Next,
Please carefully go through the guidelines and do not include the issue no's in commit messages.

So edit the current commit, on the command line :

```sh
git commit --amend
```
Change the Message to `Fix Test in Override Styles Using Important`
```sh
git push origin Test_in_Override_Styles_Using_Important_#8511 --force
```

Also next time do not enter any issue nos anywhere except for the PR description.
If you need any assistance reach out to the [Contributors Chat room](https://gitter.im/FreeCodeCamp/HelpContributors).

**Comment by hangaebal** [comment]
@raisedadead Thanks for guide.

update the pull request.

**Comment by raisedadead** [comment]
@hangaebal Thanks but the commit message is completely different now! :sweat_smile:

**Comment by raisedadead** [comment]
If you need any assistance you can hit us up in the [Contributors Chat room](https://gitter.im/FreeCodeCamp/HelpContributors).

File: client/commonFramework/step-challenge.js [hunk 1]
```diff
@@ -125,7 +125,7 @@ window.common = (function({ $, common = { init: [] }}) {
         // assume api returns string when fails
         return $el.parent()
           .find('.disabled')
-          .replaceWith('<p>' + data + '</p>');
+          .replaceWith('<p class="col-sm-4 col-xs-12">' + data + '</p>');
```
Comment (raisedadead) [inline client/commonFramework/step-challenge.js]: This is not the expected fix @hangaebal

Comment (raisedadead) [inline client/commonFramework/step-challenge.js]: @hangaebal Refer the comments left on the original thread for the expected fix, once you do, make sure you test your fix manually. By testing it.

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

:tada: Thanks for contributing @hangaebal and congratulations on your first PR submission to Open Source! :tada:

**Pull request merged**